src/SettingsModelFactory.ts

```diff
@@ -0,0 +1,69 @@
+import powerbi from "powerbi-visuals-api"
+import { parseFormattingSettingsSlice, getPropertyValue } from "./SettingsParser"
+import { FormattingSettingsCards } from "./SettingsInterfaces"
+import visuals = powerbi.visuals;
+
+export class FormattingSettingsModel {
```

Comment:
purpose here is to convert 'settings' to formattingmodel - correct?

Reply:
This class was built for two purposes:
1. convert from objects metadata dataview (received in update api) to settings object
2. convert from settings object to formattingmodel